src/App.js

```diff
@@ -1,24 +1,39 @@
-import logo from './logo.svg';
 import './App.css';
+import CountriesList from './components/CountriesList';
+import CountryDetails from './components/CountryDetails';
+import Navbar from './components/Navbar';
+import { Routes, Route } from 'react-router-dom';
+import { useEffect, useState} from 'react';
+import countriesArr from "./countries.json"
+import axios from 'axios';
 
 function App() {
+
+ 
+  const [countries, setCountries] = useState(countriesArr);
+
+ 
+  useEffect(() => {
+    axios.get("https://ih-countries-api.herokuapp.com/countries")
+    .then( response => {
+      setCountries(response.data); //save into state variable
+    })
+    .catch( e => console.log("error getting countriesfrom API", e))
+  }, [])
+
```

Comment:
and it takes a while to fetch the data, so I would add this to avoid problems because it's loading to long:
```
 if (countries.length == 0) {
    return <h1>Loading...</h1>
  }
```